hedera-mirror-rest/__tests__/specs/contracts/{id}/contract-initcode.json

```
@@ -48,6 +50,7 @@
     "obtainer_id": null,
     "permanent_removal": null,
     "proxy_account_id": null,
+    "runtime_bytecode": "0x60806040",
```

should be:
```suggestion
    "runtime_bytecode": "0x3630383036303430",
```

once the ViewModel is fixed, the response will show the above value